CHANGELOG.rst

```diff
@@ -1,5 +1,12 @@
-Unreleased
-==========
+Version 0.8rc1
+==============
+
+Released 2020-07-20.
+
+* use fissix istead of deprecated lib2to3
```

Comment:
```suggestion
* use fissix istead of deprecated lib2to3
* and other minor fixes https://github.com/python-modernize/python-modernize/compare/0.7...0.8rc1
```

Reply:
@takluyver I don't quite have the bandwidth to decode all the commits/prs into a nice changelog, but I think the fissix change is important to get into the changelog before 0.8 final

will this cop-out do for now?

Reply:
Given that I've been neglecting the changelog entirely, I can't really complain. :wink: